src/player.h

```diff
@@ -1110,6 +1110,12 @@ class Player final : public Creature, public Cylinder
 
 	const std::map<uint8_t, OpenContainer>& getOpenContainers() const { return openContainers; }
 
+	uint16_t getBaseExpGain() const { return baseExpGain; }
+	void setBaseExpGain(uint16_t value)
+	{
+		baseExpGain = std::min<uint16_t>(std::numeric_limits<uint16_t>::max(), value);
```

Comment:
```suggestion
		baseExpGain = value;
```
`std::min` is pointless as `value` can't exceed the max value of `uint16_t` because it is a `uint16_t`.

Reply:
I have almost no knowledge of C, thanks for the help.